packages/shared/src/toast/Toast.js

```diff
@@ -20,58 +25,85 @@ type State = {|
   +isVisible: boolean,
 |};
 
+type HandlerChangeEvent = {
+  +nativeEvent: {
+    +state: number,
+    ...
```

Comment:
This is cool! I didn't know that we can use it this way in Flow

Reply:
I see you are not up to date with the `@kiwicom/eslint-config` in margarita then.

It will give you an error if you try to use inexact object without `...`